Maquetado/scss/styles.scss

```diff
@@ -0,0 +1,173 @@
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+* {
+  font-family: 'Roboto', sans-serif;
+}
+
+.main-title {
+  font-size: 24px;
+  justify-content: flex-end;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.techno-title {
+  font-size: 12px;
+}
+
+.type-tool {
+  font-size: 24px;
+  position: relative;
+  transform: translateX(100%);
```

Comment:
Avoid positioning things like this, remember to always use flex box for that :smile: